src/css/components/_c-a11y-summary.scss

```diff
@@ -11,7 +11,7 @@
 	padding: 1.5rem;
 
 	@include mappy-bp(palm-large) {
-		padding: 2rem 7rem 2.75rem 7rem;
```

Comment:
Also here.

Reply:
Same as [prior explanation](https://github.com/a11yproject/a11yproject.com/pull/984/files/a442075085a0173abbc13b41848ff2affdf3df4c#r464021536) of other review.

Please advise on same direction.